wom/src/test/scala/wdl4s/wdl/values/WdlValueSpec.scala

```diff
@@ -42,28 +42,6 @@ class WdlValueSpec extends FlatSpec with Matchers {
     }
   }
 
-  val wdlFloatSpecials = Table(
-    ("wdlValue", "rawString", "validateFloat"),
-    (WdlFloat(Double.MinPositiveValue), "4.9E-324", { d: Double => d == 0.0 }),
-    (WdlFloat(Double.NaN), "NaN", { d: Double => d.isNaN }),
-    (WdlFloat(Double.MaxValue), "1.7976931348623157E308", { d: Double => d.isPosInfinity }),
-    (WdlFloat(Double.MinValue), "-1.7976931348623157E308", { d: Double => d.isNegInfinity }))
-
-  forAll(wdlFloatSpecials) { (wdlValue, rawString, validateFloat) =>
-    it should s"convert a special ${wdlValue.typeName} to/from raw string '$rawString'" in {
-      val toRawString = wdlValue.toWdlString
-      toRawString should be(rawString)
-
-      val wdlType = wdlValue.wdlType
-      val fromRawString = wdlType.fromWdlString(toRawString)
-      // Test that this is a special conversion, and is not
-      // expected to be equal after a round-trip conversion.
-      fromRawString shouldNot be(wdlValue)
-      validateFloat(fromRawString.value) should be(true)
-      fromRawString.wdlType should be(wdlType)
-    }
-  }
-
```

Comment:
Feel free to continue discussion here, but prior discussion: https://github.com/broadinstitute/wdl4s/pull/135#discussion_r126708085